From 9e01e3d9c8f89b1ef98053a64bde9beed9714029 Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Wed, 5 Feb 2014 10:06:51 +0000
Subject: [PATCH 6/6] dracut.sh: Fix variable name typo.

This caused the root_dev variable not to be set which in turn meant that
the root device was not whitelisted in 99base/module-setup.sh when injecting
compile-time devexists hooks in hostonly initrds. This ties the generated
initrd to the root fs device (typically the UUID) rather than relying solely
only the root= kernel command line.

While it is hostonly, not hardcoding e.g. UUIDs is still desirable. Any
swap partition on the host device is still added however.
---
 dracut.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index 08335d0..d693d58 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -1005,7 +1005,7 @@ if [[ $hostonly ]]; then
         _bdev=$(readlink -f "/dev/block/$_dev")
         [[ -b $_bdev ]] && _dev=$_bdev
         push host_devs $_dev
-        [[ "$_mp" == "/" ]] && root_dev="$_dev"
+        [[ "$mp" == "/" ]] && root_dev="$_dev"
         push host_devs "$_dev"
     done
 
-- 
1.8.5.5

